ipcore/generate_vhdl.m

```diff
@@ -2,9 +2,13 @@
 model = mp.modelName;
 dataplane_name = 'dataplane';
 
+%% Calculate oversampling factor from sample time
+orig_sim_mode = get_param(model, 'SimulationMode')
+set_param(model,'SimulationMode','normal')
```

Comment:
Why do we need to set the simulation mode to normal?

Reply:
`Simulink.BlockDiagram.getSampleTimes` only works when simulation mode is set to normal. Given I don't recompile and I don't think it should be automatically recompiling Matlab is almost certainly unnecessarily gating it behind that simulation mode but nothing I can do about that.